### PR TITLE
Adding staged contracts migration output JSON from Apr 17 to migratio…

### PR DESCRIPTION
**Pull request by j1010001**
related to https://github.com/onflow/cadence/issues/3265

### CODE REVIEW

**Review by jribbink — APPROVED** [review]
LGTM, thanks!

One quick quesstion: at the end of the filename, what does "103000" represent so that I can parse/interpret it correctly.

**Review by turbolent — COMMENTED** [review]
Small nit: Maybe improve the naming an make it explicit this is the _report_ for the staged contracts, not the staged contracts themselves. We might add them in the future, so maybe avoid the potential confusion.

**Comment by j1010001** [comment]
@jribbink I changed the format slightly - I added the timestamp (10-30) after the date. @turbolent I also added "report" to make it clearer, good suggestion!

File: migrations_data/staged-contracts-report--04-17-2024-10-30-testnet.json
Comment (turbolent) [inline migrations_data/staged-contracts-report--04-17-2024-10-30-testnet.json:1]: Not sure if the double dash (`--`) is intentional

Comment (jribbink) [inline migrations_data/staged-contracts-report--04-17-2024-10-30-testnet.json:1]: Maybe we could change the timestamp format to something a bit more cannonical? It's hard to know exactly what each component means or what timezone this is from.

For example, ISO 8601 (based on the UNIX timestamp I saw in a previous PR):
`staged-contracts-report-2024-04-17T20:05:50.000Z-testnet.json`
and
`staged-contracts-report-2024-04-17T20:05:50.000Z-testnet.md`

It doesn't make a difference on my end (I can parse anything as long as I know what it is), but I think this way is most obvious for somebody else looking at the file.

**Comment by turbolent** [comment]
@jribbink Please go ahead with making improvements as you deem appropriate, e.g. defining the naming scheme and renaming the existing report and the new one here, so you can continue on https://github.com/onflow/flow-cli/pull/1529 🙏

**Review by turbolent — APPROVED** [review]

**Comment by turbolent** [comment]
@jribbink The colons in the filenames are causing issues in CI, see https://github.com/onflow/cadence/actions/runs/8820926221/job/24215623175#step:10:10. Could you please remove them?

**Comment by jribbink** [comment]
> @jribbink The colons in the filenames are causing issues in CI, see https://github.com/onflow/cadence/actions/runs/8820926221/job/24215623175#step:10:10. Could you please remove/replace them?

I've made a PR to fix this 👍 https://github.com/onflow/cadence/pull/3281

**Pull request merged**